global/apps/homepage/views.py

```diff
@@ -146,11 +146,77 @@ def _get_reading_list(country_code):
 
 @cache_control(no_cache=True)
 def about(request):
+    country_code = request.META.get('HTTP_CF_IPCOUNTRY')
+    if country_code is None:
+        country_code = request.META.get(
+            'GEOIP_COUNTRY_CODE',
+            '--',
+        )
```

Comment:
I've just realised that this has nothing to do with the globa contributor list. I should separate it at least into different commits, if not different PRs.